internal/remote/aggregator_test.go

```diff
@@ -32,8 +32,9 @@ var _ = Describe("Aggregator", func() {
 	)
 
 	BeforeEach(func() {
+		colorize := true
 		reporterConfig = config.DefaultReporterConfigType{
-			NoColor:           false,
+			Colorize:          &colorize,
```

Comment:
@onsi I made `DefaultReporterConfigType.Colorize` exported for these cases, but this leads to a confusing API for non-test users of the reporter. Please let me know if it is acceptable to either drop the explicit initialization in these tests (as it doesn't seem like the tests are testing whether the output is colorized or not) or if you want a different API here.